data/tlg4034/tlg006/tlg4034.tlg006.opp-grc1.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-  <teiHeader>
+  <teiHeader xml:lang="eng">
     
     <fileDesc>
-   <titleStmt><title>Commentaria in Aristotelem Graeca</title>
+   <titleStmt><title xml:lang="lat">Commentaria in Aristotelem Graeca</title>
```

Comment:
The title should be "In Librum Quintum Ethicorum Nicomacheorum Commentarium".
Elements below that need to be corrected:
- `<idno type="filename">commentaria_22_1.xml</idno>` should be corrected to reflect the current file name.
- `<date>2014</date>` should be corrected to "2016"
- `<listBibl xml:lang="lat">` should have the `xml:lang="lat"` attribute removed.
- `<title ref="http://data.perseus.org/catalog/urn:cts:greekLit:tlg4034.tlg002.opp-grc1">In parva naturalia commentaria</title>` should be deleted.
- `<publisher>Berolini: Typis et Impensis G. Reimeri</publisher>` should have an `xml:lang="lat"` attribute. Also, the place of publication ("Berolini: ") should be deleted.